python/dgl/graphbolt/itemset.py

```diff
@@ -49,9 +49,6 @@ class requires each input itemset to be iterable.
 
     def __init__(self, items: Iterable or Tuple[Iterable]) -> None:
         if isinstance(items, tuple):
-            assert all(
-                items[0].size(0) == item.size(0) for item in items
-            ), "Size mismatch between items in tuple."
```

Comment:
So we don't need to check the tuple of tensor have the same size anymore?

Reply:
No. remove size check for tuple items as .size() is available in torch.Tensor only, not general enough as Iterable is required only for the input items.